### PR TITLE
fix related items destroy

### PR DESCRIPTION
**Pull request by andrey-abramow**

### CODE REVIEW

File: app/assets/javascripts/app/select2/related/view.coffee [hunk 1]
```diff
@@ -32,9 +32,16 @@ class @Select2Input.RelatedView extends Backbone.View
     else
       false
 
+  isDeletable: () ->
```
Comment (Rademade-review) [inline app/assets/javascripts/app/select2/related/view.coffee]: Colon assignment without proper spacing

File: lib/rademade_admin/model/adapter/data/active_record.rb [hunk 2]
```diff
@@ -33,6 +33,7 @@ def _map_relations
                   :name => name,
                   :from => @model,
                   :to => to_class,
+                  :destroyable => relation_destroyable?(relation_info),
```
Comment (Praffesor) [inline lib/rademade_admin/model/adapter/data/active_record.rb]: сделай тогда уже `destroyable` в монгоиде и сиквеле по-умолчанию false

File: lib/rademade_admin/model/adapter/data/active_record.rb [hunk 3]
```diff
@@ -47,6 +48,30 @@ def _map_relations
...
+          def _validates_association?(relation_info)
+            _model_validates?(@model,
+                             relation_info.name,
+                             ::ActiveRecord::Validations::AssociatedValidator)
+          end
+
+          def _validates_presence?(relation_info)
+            _model_validates?(relation_info.klass,
+                             relation_info.foreign_key,
+                             ::ActiveRecord::Validations::PresenceValidator)
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/active_record.rb]: Align the parameters of a method call if they span more than one line.

File: lib/rademade_admin/model/adapter/data/active_record.rb [hunk 4]
```diff
@@ -47,6 +48,30 @@ def _map_relations
...
+
+          def _validates_association?(relation_info)
+            _model_validates?(@model,
+                             relation_info.name,
+                             ::ActiveRecord::Validations::AssociatedValidator)
+          end
+
+          def _validates_presence?(relation_info)
+            _model_validates?(relation_info.klass,
+                             relation_info.foreign_key,
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/active_record.rb]: Align the parameters of a method call if they span more than one line.

File: lib/rademade_admin/model/adapter/data/active_record.rb [hunk 5]
```diff
@@ -47,6 +48,30 @@ def _map_relations
...
 
+          def _relation_destroyable?(relation_info)
+            !_validates_association?(relation_info) &&
+              !_validates_presence?(relation_info)
+          end
+
+          def _validates_association?(relation_info)
+            _model_validates?(@model,
+                             relation_info.name,
+                             ::ActiveRecord::Validations::AssociatedValidator)
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/active_record.rb]: Align the parameters of a method call if they span more than one line.

File: lib/rademade_admin/model/adapter/data/active_record.rb [hunk 6]
```diff
@@ -47,6 +48,30 @@ def _map_relations
...
           end
 
+          def _relation_destroyable?(relation_info)
+            !_validates_association?(relation_info) &&
+              !_validates_presence?(relation_info)
+          end
+
+          def _validates_association?(relation_info)
+            _model_validates?(@model,
+                             relation_info.name,
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/active_record.rb]: Align the parameters of a method call if they span more than one line.

**Pull request merged**